Packs/Censys/Integrations/CensysV2/CensysV2.yml

```diff
@@ -381,6 +477,157 @@ script:
+      type: String
+    - contextPath: Censys.IP.labels
+      description: Labels associated with the IP address.
+      type: String
+    - contextPath: Censys.IP.dns.reverse_dns.names
+      description: Reverse DNS names associated with the IP address.
+      type: String
+    - contextPath: Censys.IP.autonomous_system.country_code
+      description: The country code of the autonomous system associated with the IP address.
+      type: String
+    - contextPath: Censys.IP.autonomous_system.description
+      description: Description of the autonomous system associated with the IP address.
+      type: String
+    - contextPath: Censys.IP.autonomous_system.name
+      description: Name of the autonomous system associated with the IP address.
+      type: String
+    - contextPath: Censys.IP.autonomous_system.bgp_prefix
+      description: BGP prefix of the autonomous system associated with the IP address.
+      type: String
+    - contextPath: Censys.IP.autonomous_system.asn
+      description: Autonomous System Number (ASN) of the autonomous system associated with the IP address.
+      type: Number
+    - contextPath: Censys.IP.ip
+      description: The IP address.
+      type: String
+    - contextPath: Censys.IP.location.country
+      description: Country name of the location associated with the IP address.
+      type: String
+    - contextPath: Censys.IP.location.timezone
+      description: Timezone of the location associated with the IP address.
+      type: String
+    - contextPath: Censys.IP.location.province
+      description: Province name of the location associated with the IP address.
+      type: String
+    - contextPath: Censys.IP.location.coordinates.latitude
+      description: Latitude coordinate of the location associated with the IP address.
+      type: Number
+    - contextPath: Censys.IP.location.coordinates.longitude
+      description: Longitude coordinate of the location associated with the IP address.
+      type: Number
+    - contextPath: Censys.IP.location.continent
+      description: Continent name of the location associated with the IP address.
+      type: String
+    - contextPath: Censys.IP.location.postal_code
+      description: Postal code of the location associated with the IP address.
+      type: String
+    - contextPath: Censys.IP.location.city
+      description: City name of the location associated with the IP address.
+      type: String
+    - contextPath: Censys.IP.location.country_code
+      description: Country code of the location associated with the IP address.
+      type: String
+    - contextPath: Censys.IP.last_updated_at
+      description: The date and time when the information about the IP address was last updated.
+      type: Date
+  - name: domain
+    arguments: 
+    - name: domain
+      required: true
+      isArray: true
+      type: textArea
+    description: Return all related IPs as relationships
+    outputs:
+    - contextPath: Censys.Domain.location.postal_code
```

Comment:
how do you return this to the context output if you don't have this data for the domain?